.travis.yml

```diff
@@ -1,16 +0,0 @@
-language: php
```

Comment:
I will add TravisCI back in, not a problem. I use PHPCI because it provides so much more than just running unit test (PHPMD, PHPCS, PHPCPD, detailed code coverage reports).